app/controllers/draft_events_controller.rb

```diff
@@ -14,14 +14,14 @@ def initialize_session
   def current_round_type
     rotation = controller_store[:round_rotation]
     if rotation[Draft::ROUND_TYPE[0]] > 0
-      Draft::ROUND_TYPE[0].to_s
+      Draft::ROUND_TYPE[0].to_s.humanize
     else
-      Draft::ROUND_TYPE[1].to_s
+      Draft::ROUND_TYPE[1].to_s.humanize
     end
   end
 
   def next_round
-    if current_round_type == Draft::ROUND_TYPE[0]
+    if current_round_type == 'Cash'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.